### ISSUE TITLE
Feature Request: Option to Generate TypeScript as export interface or export type

### ISSUE THREAD

**Issue by bigmommyelven** [microsoft/node-api-dotnet#408]
First of all, thank you for your fantastic work on this library!

I would like to suggest a feature enhancement that would allow users to choose between generating TypeScript definitions as `export interface` or `export type`. Currently, the generator produces TypeScript definitions in a specific format (e.g., class), but having the flexibility to toggle between interface and type would make this tool even more versatile and better aligned with different project requirements and coding styles.

Proposed Feature
Introduce an option (eg: `[JSExport(AsInterface)]`) that allows users to specify whether TypeScript definitions should be generated as:
```
export interface SomeDto
export type SomeType
```

Thanks

**Comment by jasongin** [comment]
Maybe.. but can you easily work around this by declaring a C# interface for the class, which will then be exported as a TS interface?

**Comment by bigmommyelven** [comment]
Yes it does work. But when I called it from node env, it returns empty objects.

C#
`[JSExport]
public interface IFSFile
{
	string Name { get; set; }
	string Path { get; set; }
	string RelativeUri { get; set; }
	long Size { get; set; }
	string CreationTime { get; set; }
	string LastModified { get; set; }
	bool IsFile { get; set; }
	string Extension { get; set; }
}

public static IFSFile[] GetFiles(string path)
	=> InternalGetFiles(path);
`
JS
`
const files = FileSystem.getFiles(folderPath);
console.log(files);
`

Output
`
[IFSFile {}, IFSFile {}, ...]
`
